pkg/gardenlet/apis/config/types.go

```diff
@@ -109,6 +109,8 @@ type GardenletControllerConfiguration struct {
 	Shoot *ShootControllerConfiguration
 	// ShootCare defines the configuration of the ShootCare controller.
 	ShootCare *ShootCareControllerConfiguration
+	// ExtensionStateSynchronization defines the configuration of the ExtensionsSync controller
+	ExtensionStateSynchronization *ExtensionsStateSynchronizationControllerConfiguration
```

Comment:
Why not only

```suggestion
	ShootState *ShootStateControllerConfiguration
```

Reply:
I've renamed it to ShootState. What is the controllers naming convention in gardener. I tried to figure it out myself and thought it is that controllers are named after the resources that they are working on (watching).

Reply:
Hm, you're right, that was the (veeee..ery soft ;-)) convention so far. Indeed, `ShootState` is not too appropriate in this case as the controller is not reconciling the `ShootState` resource directly. Sorry. Maybe the old name was better... :-)